### ISSUE TITLE
Cached responses?

### ISSUE THREAD

**Issue by Bus42** [rangav/thunder-client-support#463]
**Describe the bug**

When making changes to my API, I do not see those changes reflected in subsequent requests

**To Reproduce**

1. create a request to an arbitrary endpoint with arbitrary data returned
2. make any change or even delete the entire project leaving an empty folder
3. repeat request, get the same result as the first time I made the request

**Expected behavior**

I would expect to see the changes made reflected in my request or some sort of error thrown if the request is not valid

Until this is fixed, I'm switching back to Insomnia. This bug has rendered ThunderClient basically useless.

**Platform:** 
 - OS: RHEL Linux Fedora 35
 - vscode version: 1.63.2
 - extension version: 1.11.1

**Solution:**

**Comment by rangav** [comment]
Hi @Bus42 thats for reporting the bug, is there a for me to reproduce the bug?

can you test with this url https://httpbin.org/anything
And reproduce the bug?

**Comment by rangav** [comment]
Hi @Bus42  there is not caching implemented  in Thunder Client

you can post any json data or any query parameters to the above url for testing purpose.

Or If you can share your own url and steps to reproduce the issue that will also help

**Comment by Bus42** [comment]
I didn't think ThunderClient cached anything, ergo the confusion. The issue
involves changes to the API itself. Say I have a route in my Express API
that uses a controller to access a remote database such as MongoDb (this
was my scenario). I set up the route, set up the controller, and make an
HTTP request to the API with ThunderClient. I get back the requested info
as expected. Now, if I delete that controller altogether (I went as far as
deleting my entire codebase) and repeat that request without altering it, I
get the same data as before with a 200 status code even though the
controller no longer exists and I shouldn't be getting anything back. I
would post the code, but it was developed for a client. If I have time
today, I will try to re-create it.
